Cargo.toml

```diff
@@ -19,32 +26,29 @@ ndarray = { version = "0.15", optional = true }
 num-traits = "0.2.12"
 num = "0.4"
 rand = { version = "0.8.5", default-features = false, features = ["small_rng"] }
+getrandom = { version = "*", features = ["js"] }
```

Comment:
Let me check if there are issues with enabling `js` always.

Reply:
do we need it right?
I think it is just an additional feature, shouldn't affect the base functionality

Reply:
we need an alternative for [this branch of the match](https://github.com/smartcorelib/smartcore/pull/235/files#diff-f66732050b1be4b9113c09c195087cad63579455d9f708ec0f97eacfebb3fc68R17)

Reply:
So, this feature flag only has effect for `wasm32-unknown-unknown`. Enabling this feature breakes the usage of wasm32-unknown-unknown with wasmtime and wasmer because it adds wasmbindgen to the build

Reply:
We can avoid that branch by setting a seed for those targets. In any case I think that we could use get_random but only enabling "js" feature if it is a explicitly enabled feature

Reply:
I mean,  I think that we should keep this line in the features section:

`
js = ["getrandom/js"]
`

and remove js from being enabled by default

Reply:
my bad it seems to not be working with this change

Reply:
Can we move all the changes related to rand vs getrandom to another PR? so we can merge the other changes?

Reply:
@Mec-iS I think that we shouldnt merge this PR